### PR TITLE
improve readme

### PR DESCRIPTION
**Pull request by fredboudon**
I try to update the readme so that regular questions that I have been asked are addressed. do not hesitate to improve.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,18 @@ Arguments:
 - `size_display` tuple (int, int): width and height of the canvas (minimum 400)
 - `size_world` float: extend on the 3D scene in all directions
 
+Example: 
+
+```python
+from openalea.plantgl.all import *
```
Comment (jvail) [inline README.md]: Sorry for being overly correct. I don't like the import *. Could you just import what you need? especially in a tutorial :)

File: README.md [hunk 2]
```diff
@@ -60,16 +72,30 @@ Arguments:
 - `size_display` tuple (int, int): width and height of the canvas (minimum 400)
 - `size_world` float: extend on the 3D scene in all directions
 
+Important attributes/mdethods:
+- `self.editor` : display an editor of the graphical parameters
```
Comment (jvail) [inline README.md]: Why "self" and not "lw"?

File: README.md (hunk 1 above)
Comment (fredboudon) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (fredboudon) [reply README.md]: I change this to be similar to the help(LsystemWidget). Now you have editor and get_lstring(self).

**Review by jvail — APPROVED** [review]

**Pull request merged**